karma.config.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * External dependencies
+ */
+const path = require('path');
+
+/**
+ * WordPress dependencies
+ */
+const DependencyExtractionWebpackPlugin = require('@wordpress/dependency-extraction-webpack-plugin');
+
+/**
+ * Internal dependencies
+ */
+const webpackConfigArray = require('./webpack.config');
+
+module.exports = function (config) {
+  config.set({
+    // Frameworks to use.
+    // Available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    // Base path that will be used to resolve all patterns (eg. files, exclude).
+    basePath: 'assets/src/edit-story',
+
+    // list of files / patterns to load in the browser
+    files: [{ pattern: '**/karma/**/*.js', watched: false }],
+
+    // list of files / patterns to exclude
+    exclude: ['**/test/**/*.js'],
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      '**/karma/**/*.js': ['webpack', 'sourcemap'],
+    },
+
+    webpack: webpackConfigArray
+      .filter((webpackConfig) => 'edit-story' in webpackConfig.entry)
```

Comment:
+1 for running tests closer to a production environment by using the prod version of webpack.config here.